components/yaml.rst

```diff
@@ -214,6 +228,42 @@ changed using the third argument as follows::
             foo: bar
             bar: baz
 
+Mulit-line strings as scalar blocks
+...................................
+
+To dump multi-line strings including ``\n`` character to a scalar block
+in YAML, just use the ``DUMP_MULTI_LINE_LITERAL_BLOCK`` flag::
+
+    $data = ['content' => "Lorem ipsum dolor sit amet\nconsectetur adipisicing\nelit sed do eiusmod"];
+    $dumpedData = Yaml::dump($data, 2, 4, Yaml::DUMP_MULTI_LINE_LITERAL_BLOCK);
+
+.. code-block:: yaml
+
+    content: |
+        Lorem ipsum dolor sit amet
+        consectetur adipisicing
+        elit sed do eiusmod
+
+Dump Objects as YAML Maps
+.........................
+
+You can easily dump PHP Objects as YAML map by passing ``DUMP_OBJECT_AS_MAP`` flag.
```

Comment:
This line needs to end with `::` to activate the highlight of the following PHP code block.